app/validators/answers/number_answer_validator.py

```diff
@@ -60,6 +60,8 @@ def validate_mandatory_has_no_default(self):
     def validate_value_in_limits(self):
```

Comment:
Now that this uses the schema, this should be called after `validate_referred_numeric_answer_decimals` in the main `validate()` method.

Reply:
Updated 👍